cap-2.x/common/helper-charts/external-dns.tf

```diff
@@ -1,17 +1,34 @@
 resource "helm_release" "external-dns" {
-  name  = "cap-external-dns"
-  repository = "https://marketplace.azurecr.io/helm/v1/repo"
```

Comment:
Bitnami explicitly recommends pulling from the Azure Marketplace when using with AKS:

https://bitnami.com/stack/external-dns/helm

Reply:
It's for efficiency as Azure Container Registry is geo-replicated so deployment on AKS would benefit from quicker image pulls. With our image sizes, this is a non-issue and we don't want to introduce conditional logic in the common chart for the registry url.